aspnetcore/blazor/fundamentals/startup.md

```diff
@@ -72,6 +90,94 @@ For an examples of JS initializers, see the following resources:
 
 [!INCLUDE[](~/includes/aspnetcore-repo-ref-source-links.md)]
 
+### Ensure libraries are loaded in a specific order
+
+Append custom scripts to the `<head>` in `beforeStart` and `afterStarted` in the order that they should load.
+
+The following example loads `script1.js` before `script2.js` and `script3.js` before `script4.js`:
+
+```javascript
+export function beforeStart(options, extensions) {
+    var customScript1 = document.createElement('script');
+    customScript1.setAttribute('src', 'script1.js');
+    document.head.appendChild(customScript1);
+
+    var customScript2 = document.createElement('script');
+    customScript2.setAttribute('src', 'script2.js');
+    document.head.appendChild(customScript2);
+}
+
+export function afterStarted(blazor) {
+    var customScript1 = document.createElement('script');
+    customScript1.setAttribute('src', 'script3.js');
+    document.head.appendChild(customScript1);
+
+    var customScript2 = document.createElement('script');
+    customScript2.setAttribute('src', 'script4.js');
+    document.head.appendChild(customScript2);
+}
+```
+
+### Import additional modules
+
+Use top-level [`import`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/import) statements in the JS initializers file (`*.lib.module.js`) to import additional modules.
+
+`additionalModule.js`:
+
+```javascript
+export function logMessage() {
+  console.log('logMessage is logging');
+}
+```
+
+```javascript
+import { logMessage } from "/additionalModule.js";
+
+export function beforeStart(options, extensions) {
+  ...
+
+  logMessage();
+}
+```
+
+For browser compatibility, see [Can I use: JavaScript statement: import](https://caniuse.com/?search=JavaScript%20statement%3A%20import).
+
+[Dynamic import with the `import()` operator](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/import) is supported with ASP.NET Core and Blazor:
+
+```javascript
+import("/additionalModule.js");
+```
+
+For browser compatibility, see [Can I use: JavaScript modules: dynamic import](https://caniuse.com/es6-module-dynamic-import).
+
+### Export additional JavaScript in the JavaScript initializers file
+
+Because the JS initializers file (`*.lib.module.js`) is an ordinary ES6 module, exporting additional JS side-by-side with `beforeStart` and `afterStarted` is supported:
+
+```javascript
+export function beforeStart(options, extensions) {
+  ...
+}
+
+export function afterStarted(blazor) {
+  ...
+}
+
+export function customFunction() {
+  ...
+}
+```
+
+### Import map
+
+[Import maps](https://developer.mozilla.org/docs/Web/HTML/Element/script/type/importmap) are supported by ASP.NET Core and Blazor.
+
+For browser compatibility, see [Can I use: import maps](https://caniuse.com/import-maps).
+
+### Module extension (`.js`/`.mjs`)
+
+Both `.js` and `.mjs` module extensions are supported by ASP.NET Core and Blazor, which are sent with a MIME type of `text/javascript`.
```

Comment:
Have customers shown any confusion about these features? I'm wondering if it's even worth pointing out that they work, since I think they should automatically be expected to work in any web app.

Reply:
No, you're correct. Let's kill 💀 off any content that isn't helpful.

Reply:
I wonder if, rather than getting into specifics, it would be sufficient to say something like:

> JavaScript module loading works the same way in Blazor as it does for other types of web apps, and you're free to customize how modules are defined in your app. See https://developer.mozilla.org/docs/Web/JavaScript/Guide/Modules for a guide on how to use JavaScript modules.

Or, do you think it would be worth it to include our own guidance? We could also do both - briefly summarize some ways to define/use JS modules and then link to the MDN doc for more info.

Reply:
Well, so much for getting back to this Thursday/Friday. Moving across Florida has been a crazy whirlwind of activity 😫. I'm good now tho ... but I am ***living in a SEA of BOXES!*** 😆 

WRT the text/approach ... yes ... that's cool. I'll use that in the JS interop modules INCLUDE that I've sprinkled in where appropriate. I'll remove this `.mjs` bit.